src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] ballArray = new Ball[] {
+                lottery.getRandomBall(),
+                lottery.getRandomBall(),
+                lottery.getRandomBall()};
+
+        for (Ball currentBall : ballArray) {
+            System.out.println(currentBall.toString());
```

Comment:
```suggestion
            System.out.println(currentBall);
```
toString is called by default